### ISSUE TITLE
Link in tooltip

### ISSUE THREAD

**Issue by ShKlinkenberg** [AdeelK93/collapsibleTree#65]
Hi @AdeelK93 

I want to create a collapsibleTreeNetwork with a link in the tooltip. I have forked your repo so I could change the mouseout function for hiding the tooltip. I now have some time to actually go to the link. Though the link is not clickable. My three can be found [here](https://werkgroep-toetsen-op-afstand.github.io/Kennistoetsen-over-Toetsing/index.html), and the code [here](https://github.com/Werkgroep-Toetsen-Op-Afstand/Kennistoetsen-over-Toetsing).

I have [read this article](http://bl.ocks.org/d3noob/c37cb8e630aaef7df30d) so I would think it would be possible in D3. Though while looking at the code i do not see a difference in approach. 

Do you have any suggestions to get this to work?

**Comment by ShKlinkenberg** [comment]
I managed to solve it. Two thinks needed to be change

1. [Mouseout needed to be deleted](https://github.com/AdeelK93/collapsibleTree/blob/c1901a4ffac3001a06a64e44e0417d17447a0222/inst/htmlwidgets/collapsibleTree.js#L64) to allow the permanent visibility of the tooltip.
2. In the CSS for the htmlwidget, [collapsibleTree.css](https://github.com/AdeelK93/collapsibleTree/blob/master/inst/htmlwidgets/lib/collapsibleTree.css), the [pointer-event](https://github.com/AdeelK93/collapsibleTree/blob/c1901a4ffac3001a06a64e44e0417d17447a0222/inst/htmlwidgets/lib/collapsibleTree.css#L29) needed to be deleted.

It would be cool to add tooltipPermanent as an additional argument in the collapsibleTree function.

**Comment by rlderi** [comment]
@ShKlinkenberg thank you, I have tried to do this previously and failed.

**Comment by askarlupka** [comment]
@ShKlinkenberg I'm wanting to add a tooltipHTML to my tree, but one that is made with the collapsibleTree() function instead of collapsibleTreeNetwork() function. Currently, collapsibleTree() only allows for numeric tooltips, unless it's in the flat network format and paired with collapsibleTreeNetwork(). However, my nodes do not have unique names and are in a nested lists. 

Any suggestions on trying to get strings/HTML to show up for collapsibleTree()? I'm attempting to either make a dataframe that can be read into collapsibleTreeNetwork, or change the underlying code for collapsibleTree to allow for character class arguements for tooltip. But I'm not sure which will be doable.

**Comment by askarlupka** [comment]
@ShKlinkenberg I think I figured it out. For future reference I changed the code for collapsibleTree.Node in line 18 from 

` if(!is.null(tooltipHtml)) if(!(tooltipHtml %in% df$attributes)) stop("tooltipHtml column name is incorrect")`
to 
` if(!is.null(tooltipHtml)) if(!(tooltipHtml %in% df$attributesAll)) stop("tooltipHtml column name is incorrect")`

So far it seems to be working as expected.

**Comment by AdeelK93** [comment]
@askarlupka good catch. I know I haven't been great at maintaining this, but if you want to PR that, I can put out a release
